### PR TITLE
Feature namelaowang

### PR DESCRIPTION
**Pull request by namelaowang**
https://github.com/TestUpCommunity/TUGithubAPI/issues/558
https://github.com/TestUpCommunity/TUGithubAPI/issues/559
https://github.com/TestUpCommunity/TUGithubAPI/issues/560
https://github.com/TestUpCommunity/TUGithubAPI/issues/561

### CODE REVIEW

**Comment by namelaowang** [comment]
追加https://github.com/TestUpCommunity/TUGithubAPI/issues/563

**Comment by namelaowang** [comment]
完成https://github.com/TestUpCommunity/TUGithubAPI/issues/502

**Comment by namelaowang** [comment]
完成 https://github.com/TestUpCommunity/TUGithubAPI/issues/131

File: api/apps/installactions.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from core.rest_client import RestClient
+
+
+class Installation(RestClient):
+    def list_repositories(self, **kwargs):
+        """
+        https://developer.github.com/v3/apps/installations/#list-repositories
+        :param kwargs:
```
Comment (zhangting85) [inline api/apps/installactions.py:8]: 要么不写param，要么写全，写个参数名字不加解释是不对的。

**Pull request merged**